string control_mode
bool motors_on
float64 roll_cmd # deg
float64 pitch_cmd # deg
float64 yaw_cmd # deg
float64 yaw_rate_cmd # deg
float64 alt_cmd # m